### PR TITLE
Refactor frontend architecture

### PR DESCRIPTION
**Pull request by cveneziani**
I tried to have (even) more structured stylesheets.
- Settings
- Tools (mixins / functions)
- Vendors
- Generic (ground zero styles / reset / normalize)
- Base
- Objects
- Components (for now also Objects :grimacing:)
- Layout
- Page

Can you tell me **quickly** what you think about it?

### CODE REVIEW

File: source/assets/stylesheets/pages/home/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    background: none;
+    font-weight: $font-weight-bold;
+  }
+
+  .top-bar-section a {
+    font-weight: $font-weight-bold;
+  }
+}
+
+.p-home-learn-more-link {
```
Comment (flexbox) [inline source/assets/stylesheets/pages/home/_hero.scss:62]: Hummm, maybe you can try this ?

```
.p-home-learn-more-link {
  display: flex;
  .icon {
    justify-content: center;
    align-items: center;
    ...
  }
}
```

**Comment by flexbox** [comment]
Great! 

According to me you can simplify your system 
Here is an example — I have only 3 folders
https://gist.github.com/b84270a26fbdde2802a109b51a4b85b9

http://patternlab.io/

And maybe you don't need extras caracters `c-`, `p-`, `l-`, just to win some time on coding :)

**Comment by flexbox** [comment]
Review done!

File: source/assets/stylesheets/pages/home/_hero.scss (hunk 1 above)
Comment (cveneziani) [inline source/assets/stylesheets/pages/home/_hero.scss:62]: I tried something. I think it wasn't what you meant but kind of works.

**Comment by cveneziani** [comment]
I agree with you. I could simply have less folders but I did try to have more structure in order to be less confused. 

The same for the prefixes `c-`, `l-`, `p-`. It helps me to know where is my stuff.

**Comment by cveneziani** [comment]
Fix done.

File: source/assets/stylesheets/pages/home/_hero.scss (hunk 1 above)
Comment (flexbox) [inline source/assets/stylesheets/pages/home/_hero.scss:62]: Great !

**Comment by flexbox** [comment]
You can proceed to merge 🎉

**Comment by cveneziani** [comment]
Yay! Thanks! :tada:

File: source/assets/stylesheets/components/_typography.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.emperor, %emperor {
```
Comment (cveneziani) [inline source/assets/stylesheets/components/_typography.scss:1]: BTW have you seen this changes?

I translated these ones:
- `vador` :fr:  into `vader` :gb: 
- `acolyte` :fr:  into `sidekick` :gb: 

And added the `human` for `h6`.

You might wanna update your styles & blog post.

**Pull request merged**